parquet/src/main/scala/monix/connect/parquet/ParquetSubscriberT.scala

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2020-2020 by The Monix Connect Project Developers.
+ * See the project homepage at: https://connect.monix.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monix.connect.parquet
+
+import monix.eval.Task
+import monix.execution.cancelables.AssignableCancelable
+import monix.execution.internal.Platform
+import monix.execution.{Ack, Callback, Scheduler}
+import monix.reactive.Consumer
+import monix.reactive.observers.Subscriber
+import org.apache.parquet.hadoop.ParquetWriter
+
+import scala.concurrent.Future
+import scala.util.control.NonFatal
+
+/**
+  * A sink that writes each element of [[T]] passed, into a single parquet file.
+  *
+  * @param parquetWriter The apache hadoop generic implementation of a parquet writer.
+  * @tparam T Represents the type of the elements that will be written into the parquet file.
+  */
+private[parquet] class ParquetSubscriberT[T](parquetWriter: Task[ParquetWriter[T]]) extends Consumer[T, Long] {
```

Comment:
`T` seems like a `Transformer` but it's internal code so it doesn't really matter :D

Reply:
Yeah I know, but it is also good to represent `Task`, should I use `L` like on the observable or maybe `F`? 
please say if you have a better one

Reply:
`L` are for methods that "fold" `Observable` into `Task` and `F` is for `F[_]` IIRC

Maybe `Safe` or `Eval` somewhere but that's just bike-shedding the code that users won't even see :P